packages/framework/azure-client/src/interfaces.ts

```diff
@@ -30,6 +30,15 @@ export interface AzureClientProps {
     readonly logger?: ITelemetryBaseLogger,
 }
 
+export interface AzureContainerVersion {
+    // Version ID
```

Comment:
Nit: you need to use these comments to be picked up by the doc tool

```suggestion
    /**
     * Version ID
     **/
```

Reply:
Do these ids follow some sort of ordering?  might be good to include some notes on the format somewhere in the documentation

Reply:
also are these ids equivalent to the version argument from the other recreateContainer API?  might be good to use a similar name there e.g. versionId

Reply:
Here we do not assume any ordering. `AzureContainerVersion` gives you additional meta around the version information. I was debating here. Potentially we could use `AzureContainerVersion` to pass version information to recreateContainer API, so we can ensure we know how to interpret the version. Let's say at some point later, we switch to mechanism that is not snapshot based. With that, we may end up having different version type.  It may be rare occurrence where client uses get versions on one azure client, saves it, and then uses the container version info to recreate container through another azure client (that may be relying on different mechanism).

More thoughts on versioning in the upcoming comment.